pkg/kubelet/metrics/metrics.go

```diff
@@ -515,7 +515,9 @@ func SetActiveConfig(source *corev1.NodeConfigSource) error {
 		return err
 	}
 	// clean up the old timeseries (WithLabelValues creates a new one for each distinct label set)
-	ActiveConfig.Delete(activeConfigLabels)
+	if ret := ActiveConfig.Delete(activeConfigLabels); !ret {
+		klog.Warningf("unable to delete active config labels")
```

Comment:
Nit: Are there any actions that we want to recommend the cluster admin to take when they see this message?

Reply:
Currently I don't have recommend to add.